app/views/kudos/show.html.erb

```diff
@@ -26,7 +26,11 @@
   </p>
 
   <% if kudo.giver == current_employee %>
-    <%= link_to 'Edit', edit_kudo_path(kudo), class:"btn btn-dark" %>
+   <% if policy(kudo).update? %>
```

Comment:
Here is the place where we can benefit from checking current employees in policy (check comment in `kudo_policy.rb')
or we can check this with only one if
```ruby
<% if kudo.giver == current_employee && policy(kudo).update? %>
```

but I'm leaning toward the first solution (all checks in policy). :)